specification/iothub/resource-manager/Microsoft.Devices/preview/2018-12-01-preview/iothub.json

```diff
@@ -1978,9 +1978,49 @@
             "name": "Capabilities",
             "modelAsString": true
           }
+        },
+        "locations": {
+          "description": "Primary and secondary location for iot hub",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IotHubLocationInfo"
+          }
         }
       }
     },
+    "IotHubLocationInfo": {
+      "description": "Locations details for iot hub.",
+      "type": "object",
+      "properties": {
+        "location": {
+          "description": "Azure Geo Regions",
+          "type": "string"
+        },
+        "role": {
+          "description": "Specific Role assigned to this location",
+          "enum": [
+            "Primary",
+            "StandbySecondary",
+            "ReplicatingSecondary",
+            "RestoringPrimary",
+            "Unprovisioned"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "IotHubReplicaRole",
+            "modelAsString": true
+          }
+        },
+        "scaleUnitName": {
```

Comment:
Sorry but I've never seen this return from GET IoT Hub before. In the `locations` property for my IoT Hub I see only

```json
    "locations": [
      {
        "location": "West US",
        "role": "primary"
      },
      {
        "location": "East US",
        "role": "secondary"
      }
    ],
```

In the description below can you add when this would show up?

Reply:
updated